### DISCUSSION TITLE
Hide my karma score

### DISCUSSION THREAD

**Discussion by touhf** [Docile-Alligator/Infinity-For-Reddit#1025] [Completed Ideas] [answered]
Toggle in settings to hide karma score in profile and menu

**Comment by anomopoda** [comment]
Would also love to be able to hide my karma in the navigation sidebar!

![photo_2022-09-26 17 57 27](https://user-images.githubusercontent.com/78587131/192259856-807e58b3-b64d-4674-8d74-beaf4afece72.jpeg)

**Comment by patrickdemers6** [comment]
This shouldn't be too difficult of a change and would reduce unnecessary clutter for users that view it as clutter.

@Docile-Alligator or @Wladefant, is this something you would be okay having implemented? I can work on it if so.

**Reply by Docile-Alligator** [reply]
Yes. It's ok. Thank you.

**Comment by Wladefant** [comment, marked as answer]
Implementation by @patrickdemers6 #1168
